### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by molekilla**

### CODE REVIEW

File: src/Components/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            this.toggleErrorDialog(true, 'Error', text)
+        }
+    }
+
+
+    signUpMnemonic = () => {
+        try {
+            const vetWallet = Wallet.fromMnemonic(this.state.mnemonic, "m/44'/818'/0'/0/0")
+            const wallet = Wallet.fromMnemonic(this.state.mnemonic)
+            console.log('signUpMnemonic', vetWallet, wallet)
```
Comment (decent-dev) [inline src/Components/Login/Login.jsx]: We could clean this up from here as well as in `onPassphraseListener()` in `NewWallet.jsx`

File: src/Components/Dashboard/DashboardAppBar.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    address,
+    ethBalance,
+    vthoBalance,
+    currency,
+    isLoading,
+    onMenuToggle,
+    onAddressCopyListener
+}) {
+    return (
+        <div>
```
Comment (zubietaroberto) [inline src/Components/Dashboard/DashboardAppBar.jsx]: Why extra div?

File: src/Components/Dashboard/Dialogs/AboutDialog.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    </Typography>
+                </DialogContentText>
+            </DialogContent>
+            <DialogActions>
+                <Button
+                    variant="contained"
+                    onClick={onCloseListener}
+                    color="primary"
+                >
+                    OK
```
Comment (zubietaroberto) [inline src/Components/Dashboard/Dialogs/AboutDialog.jsx]: Non translated

File: README.md [hunk 4]
```diff
@@ -45,11 +50,20 @@ A production build will be created in the build_webpack folder.
...
-Make sure you run **npm run build** and have the build_webpack folder in the wallet's root 
+Make sure you run `yarn run build` and have the `build_webpack` folder in the wallet's root 
 directory. Once available, simply call 
 
-```
-npm run dist
-```
+- Windows `yarn dist:windows`
+- Linux `yarn dist:linux`
+- MacOs `yarn dist:windows`
```
[the remark this answers is not in the document]
Comment (bradleySuira) [reply README.md]: fixed DBET-769_realse_builder_and_auto_updater

File: README.md [hunk 5]
```diff
@@ -1,8 +1,14 @@
-# DBET Web/Desktop Wallet
+# DBET Web/Desktop Wallet - v1.0.0 - Vechain / Ethereum Version
```
[the remark this answers is not in the document]
Comment (bradleySuira) [reply README.md]: changed on DBET-769_realse_builder_and_auto_updater

**Pull request merged**